doc/source/ray-core/ray-dashboard.rst

```diff
@@ -106,6 +106,21 @@ The log viewer provides various search functionality to help find the log messag
 Advanced Usage
 --------------
 
+Viewing built-in dashboard API metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The API server that serves the dashboard also emits basic prometheus metrics for API call count and API call duration.
+
+`dashboard_api_requests_count_requests_total`: Collects the total count of requests. This is tagged by endpoint, method, and http_status.
+
+`dashboard_api_requests_duration_seconds_bucket`: Collects the duration of requests. This is tagged by endpoint and method.
```

Comment:
Actually can you describe exactly where these variables are used and when you would use them? I gather they are prom metrics, where can I read more?